src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/OptionalIntervalArgumentCollection.java

```diff
@@ -15,7 +15,7 @@
     private static final long serialVersionUID = 1L;
 
     @Argument(fullName = StandardArgumentDefinitions.INTERVALS_LONG_NAME, shortName = StandardArgumentDefinitions.INTERVALS_SHORT_NAME,
-            doc = "One or more genomic intervals over which to operate", optional = true)
+            suppressFileExpansion = true, doc = "One or more genomic intervals over which to operate", optional = true)
```

Comment:
Add comment here explaining why file expansion is suppressed for this arg. The comment should be a non-javadoc comment (eg., //) so that it won't be picked up by the doc generation code.

Reply:
done.